milestone-1-survey.md

```diff
@@ -0,0 +1,92 @@
+# Axon Framework 5 M1 Survey
+
+We, the Axon Framework team, are very happy to announce the first milestone of Axon Framework 5.
+While not a complete release, we are excited to share the progress we've made so far.
+
+The biggest motivator for a Milestone release is to gather feedback from the community.
+So we encourage you to try it out and share your thoughts with us.
+
+## Scope
+
+This milestone release is focused on the following areas:
+
+- *Asynchronous core*: The core of Axon Framework has been completely rewritten to be asynchronous by default.
+- *Configuration*: The configuration of Axon Framework has been simplified and made more flexible. Support for
+  decorators has been added, and you can create encapsulated modules with their own partial configuration.
+- *Dynamic Consistency Boundary*: We are breaking free from the aggregate concept. Entities now allow a flexible
+  boundaries, making it possible to combine events from different tags. This greatly reduces the need to use sagas, and
+  makes code flexible for whatever requirements pop up in the future.
+
+We plan to release at least two more milestones releases before the final release of Axon Framework 5. This milestone
+release does not contain, for example:
+
+- Spring Boot support
+- Redesigned event processors
+- TODO: Base on roadmap?
+
+## Try it out
+
+We encourage you to try out the milestone release and share your feedback with us.
+You can follow the instructions in
+the [Getting Started with Axon Framework 5 guide](https://docs.axoniq.io/axon-framework/todo-fill-in-url) to get
```

Comment:
Ah, yes, we need to figure out how we're going to get the getting started out... I've added this pointer to our session calendar for tomorrow.